src/packages/admin-ui-components/src/detail-panel/graphql.ts

```diff
@@ -7,7 +7,7 @@ export const generateUpdateEntityMutation = (
 ) => gql`
     mutation updateEntity ($input: ${entity.name}UpdateInput!){
       update${entity.name} (input: $input) {
-        ${generateGqlSelectForEntityFields(entity, entityByType)}
+        ${generateGqlSelectForEntityFields(entity.fields.filter((field) => !field.hideInDetailForm), entityByType)}
```

Comment:
There is a bug in generateGqlSelectForEntityFields right now when items are enabled for the list but not for edit or vice versa. This solution requires the caller to know exactly what they are supposed to filter on, based on if they are fetching for a list or edit scenario. It could also be implemented as extra boolean parameters passed to generateGqlSelectForEntityFields, or new properties on the entity class, like `entity.fieldsForLists` and `entity.fieldsForEditing`